### DISCUSSION TITLE
What exactly is the use of this?

### DISCUSSION THREAD

**Discussion by Espnandro** [QL-Win/QuickLook#1373] [General]
I can open an image or video in less than a second, and to see details/size of a file I can open properties faster than I can highlight the file and press spacebar. 

I do not mean this in "why would anyone use this" kind of way, I'm genually asking because I've seen people recommend this program, and after trying it I feel as though I'm faster without it. I do see people praise QuickLook on reddit and I'm wondering if it's just people who are used to it from Mac OS or if it's really useful. To me it just seems like a less detailed file properties menu bound to spacebar, but I can open the properties of a file much faster than I can preview it. 

Maybe on a HDD or slower PC opening an image is slower than opening a preview but on my machine images and videos open instantly. Again, I'm sorry if this reads condescending, I really am asking genually because many people swear this is a great tool.

**Comment by tomhundt** [comment]
1. It doesn't show the properties; it shows the *contents*. 
2. Simplicity: Maybe you're using IrfanView or Sumatra to preview images and PDFs, both of which are really fast.  It's still more keystrokes, though, to rummage through a bunch of files in File Explorer using up/down arrows than up arrow / Enter (to open) / Esc (to close again).  It's just way quicker to hit the spacebar.  And if you have a folder of dissimilar files (documents, images, zipfiles...) it's nice to have _one_ way of rummaging through them.
3. Sort order:  Also, while doing (for example) PageUp/PageDn in Irfanview is really quick to view a folder full of images, the sort order and files included are going to be different from the list displayed in File Explorer and that might be disconcerting at times.  E.g., if Irfanview is using filename sort and Explorer modtime sort, those are often different.
4. Speed: Many programs aren't so fast to spin up (hello, Office/OpenOffice) so having an alternative way of previewing ends up being much, much faster.

**Comment by L-K-M** [comment]
It's low-friction, fast, and non-modal. I don't want to open the file and then close it; I just want to see its contents. I often open Quick Look in a list of images and then move between images using the arrow key while it's open. When I'm done, I don't even have to close Quick Look; I just move on.

**Comment by antodc** [comment]
I used MacOS for a few years in the past and the preview function is probably the one I missed most in Windows. I only discovered this terrific software a few days ago. It is really great! It makes my work considerably faster and more pleasant. Many kudos and heartfelt thanks to the developer(s).
